README.md

```diff
@@ -1,7 +0,0 @@
-# Performance Reports
```

Comment:
You did delete the other copied README, but it seems you have also deleted the original one!